### DISCUSSION TITLE
Thank You!

### DISCUSSION THREAD

**Discussion by Splyth** [dyc3/steamguard-cli#375] [General]
### Description

Hey I just wanted to publicly say thank you for the work you do on this repo! I was extremely concerned I wouldn’t be able to automate my game’s Steam deployment because 2fa is required to push content to steam.

and I wanted to containerize the process so I couldn’t just stay signed in (containers don’t tend to stick around)

and here you are solving my showstopper of an issue neatly and easily! 

You’re the best!
Scott M.

**Comment by dyc3** [comment]
Glad I could help!
